rust/operator/src/pod_svc_controller.rs

```diff
@@ -11,10 +11,10 @@ use stackable_operator::{
 use std::{sync::Arc, time::Duration};
 use strum::{EnumDiscriminants, IntoStaticStr};
 
+pub const POD_SERVICE_CONTROLLER_NAME: &str = "pod-service";
```

Comment:
The other controller is called "kafkacluster" (which I also find confusing). Maybe rename both for consistency?

Reply:
This was just using the older `RESOURCE_SCOPE` naming. The controller responsible for the pod-service?